kubernetes/ipa-ca-configmap.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
```

Comment:
Technically, yes, but it's kind of bespoke in the sense that this gets replicated into the monitoring and tooling namespaces (see the reflector annotations below), I felt it was bespoke enough that it's fine to keep separate intentionally (similar to cluster wide secrets).

If in default, it's not immediately clear why it would be in the monitoring and tooling namespaces as well if you observe the cluster.

Reply:
Humm I'm not strongly opinionated on the topic anyways.

I was mostly wondering